schema/revert/tables/sector_001.sql

```diff
@@ -0,0 +1,8 @@
+-- Revert cif:tables/sector_001 from pg
+
+begin;
+
+delete from cif.sector where sector_name = 'Waste';
+delete from cif.sector where sector_name = 'Manufacturing';
```

Comment:
Just a suggestion, we can make this one-liner:
```
delete from cif.sector where sector_name in ('Waste', 'Manufacturing'):
```

Reply:
done - and committed as a feat :)